### PR TITLE
Javascript

### PR DESCRIPTION
**Pull request by Aparnakrishnan-Vonnue**

### CODE REVIEW

File: Javascript-tutorial/People-counter-app/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//document.getElementById("count-el").innerText=5
+
```
Comment (akshaysama) [inline Javascript-tutorial/People-counter-app/index.js:2]: why is the js file commented out, will this work without js ?

File: Javascript-tutorial/Shopping-site/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="card">
+            <img src="assets/images/nike.webp" alt="nike">
+            <p>Nike Shoe</p>
+            <button id="purchase" onclick="purchase()">Purchase-$149</button>
+            <p id="error"></p>
```
Comment (akshaysama) [inline Javascript-tutorial/Shopping-site/index.html:11]: why is the id same for various elements? the id attribute should be unique within the entire document

File: Javascript-tutorial/Shopping-site/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            <script src="index.js"></script>
+        </div>
+        <div class="card">
+            <img src="assets/images/nike.webp" alt="nike">
+            <p>Nike Shoe</p>
+            <button id="purchase" onclick="purchase()">Purchase-$149</button>
+            <p id="error"></p>
```
Comment (akshaysama) [inline Javascript-tutorial/Shopping-site/index.html:18]: https://www.w3schools.com/html/html_id.asp refer this

File: Javascript-tutorial/Shopping-site/index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            <script src="index.js"></script>
+        </div>
+        <div class="card">
+            <img src="assets/images/nike.webp" alt="nike">
+            <p>Nike Shoe</p>
+            <button id="purchase" onclick=purchase()>Purchase-$149</button>
+            <p id="error"></p>
```
Comment (akshaysama) [inline Javascript-tutorial/Shopping-site/index.html:32]: ith copy ahno ??

File: Javascript-tutorial/Shopping-site/index.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+
+let errorParagraph = document.getElementById("error")
```
Comment (akshaysama) [inline Javascript-tutorial/Shopping-site/index.js:2]: does this work when you have given multiple elements with same id's

Comment (akshaysama) [reply Javascript-tutorial/Shopping-site/index.js:2]: you can use call elements by their class name or can use the queryselector all method if you need to call multiple elements with same class names, you will come accross this on going forward with the tutorial

File: Javascript-tutorial/blackjack game/index.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```
Comment (akshaysama) [inline Javascript-tutorial/blackjack game/index.html:1]: how is the html file connected with the script.js file

**Comment by akshaysama** [comment]
address review comments

File: Javascript-tutorial/Shopping-site/index.html (hunk 4 above)
Comment (Aparnakrishnan-Vonnue) [reply Javascript-tutorial/Shopping-site/index.html:32]: tutorial aan javascriptinte

**Pull request merged**